flexmeasures/data/migrations/versions/ac5e340cccea_drop_constraint_generic_asset_name_account_id.py

```diff
@@ -0,0 +1,36 @@
+"""Drop generic_asset_name_account_id_key constraint
```

Comment:
This (and correspondingly, the file name) should be updated to mention we are just changing the scope of this constraint (from account to parent asset).

Reply:
Good point!